src/components/elements/MapPolygonPanel/VersionInformation.tsx

```diff
@@ -0,0 +1,116 @@
+import classNames from "classnames";
+import { Dispatch, SetStateAction } from "react";
+
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+import ModalConfirm from "@/components/extensive/Modal/ModalConfirm";
+import { useModalContext } from "@/context/modal.provider";
+
+import Menu from "../Menu/Menu";
+import { MENU_PLACEMENT_RIGHT_BOTTOM } from "../Menu/MenuVariant";
+import Text from "../Text/Text";
+
+const VersionInformation = ({ setPreviewVersion }: { setPreviewVersion: Dispatch<SetStateAction<boolean>> }) => {
+  const { openModal, closeModal } = useModalContext();
+  const openFormModalHandlerConfirm = () => {
+    openModal(
+      <ModalConfirm
+        title={"Confirm Version Delete?"}
+        content="Do you want to delete this version?"
+        onClose={closeModal}
+        onConfirm={() => {}}
+      />
+    );
+  };
+  const itemsPrimaryMenu = [
+    {
+      id: "1",
+      render: () => (
+        <Text variant="text-14-semibold" className="flex items-center">
+          <Icon name={IconNames.SEARCH} className="h-4 w-4 lg:h-5 lg:w-5" />
+          &nbsp; Preview Version
+        </Text>
+      ),
+      onClick: () => setPreviewVersion(true)
+    },
+    {
+      id: "2",
+      render: () => (
+        <Text variant="text-14-semibold" className="flex items-center">
+          <Icon name={IconNames.TRASH_PA} className="h-5 w-4 lg:h-6 lg:w-6 " />
+          &nbsp; Delete Version
+        </Text>
+      ),
+      onClick: () => openFormModalHandlerConfirm()
+    }
+  ];
+  const data = [
+    {
+      title: "ID Wenguru v4",
+      date: "Feb 12, 24",
+      current: "No"
+    },
+    {
+      title: "ID Wenguru v3",
+      date: "Feb 11, 24",
+      current: "Yes"
+    },
+    {
+      title: "ID Wenguru v2",
+      date: "Feb 10, 24",
+      current: "No"
+    },
+    {
+      title: "ID Wenguru v1",
+      date: "Feb 8, 24",
+      current: "No"
+    },
+    {
+      title: "ID Wenguru v1",
+      date: "Feb 6, 24",
+      current: "No"
+    }
+  ];
```

Comment:
Please remove all mocked data in component code.

Reply:
done